### PR TITLE
List directories

### PR DESCRIPTION
**Pull request by ryandm**
I ran into the problem of wanting to see what was in directories without going back to shell, so I could explore the code more easily from the browser.

This isn't the nicest solution --- it replaces the 400 message with a directory listing in the case the edit path points to a directory --- but it lets me navigate ok. It's also a tiny way to get started with thinking about this project.

...written in coffee-table of course :)

### CODE REVIEW

**Comment by jimtla** [comment]
Hmm - you may have to merge master back into this in order to get a sane diff (it's currently showing all of my code in the diff).

**Comment by ryandm** [comment]
done

File: src/edit.coffee [hunk 1]
```diff
@@ -25,6 +25,15 @@ module.exports = (app) ->
                         res.send 400, err
                 else
                     res.render 'edit', {file, contents}
+            fs.stat target, (err, stat) ->
+                if not err and stat.isDirectory()
+                    fs.readdir target, (err, files) ->
+                        if err
+                            readFile()
+                        else
+                            res.send 400, files
```
Comment (jimtla) [inline src/edit.coffee]: This should probably be a 200 response instead of a 400

**Comment by jimtla** [comment]
one minor comment, +1

**Pull request merged**